packages/deploy/README.md

```diff
@@ -30,7 +30,12 @@ const { DeploymentClient } = require('platform-deploy-client');
 const client = new DeploymentClient({ apiKey: API_KEY, apiSecret: API_SECRET });
 ```
 
-### Deployment Config
+```js
+const { UpgradeClient } = require('platform-deploy-client');
+const client = new UpgradeClient({ apiKey: API_KEY, apiSecret: API_SECRET });
+```
+
+### Deployment Config (DEPRECATED)
```

Comment:
maybe we could specify after which version it is deprecated

Reply:
or maybe our documentation tool has some versioning features?

Reply:
Since the deploy is tagged as Alpha, it should be fine to remove this from the documentation entirely. This is deprecated since it got replaced with approval processes..